cluster/gce/config-common.sh

```diff
@@ -158,7 +158,7 @@ export WINDOWS_BOOTSTRAP_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubelet.bootstrap-k
 # Path for kube-proxy kubeconfig file on Windows nodes.
 export WINDOWS_KUBEPROXY_KUBECONFIG_FILE="${WINDOWS_K8S_DIR}\kubeproxy.kubeconfig"
 # Pause container image for Windows container.
-export WINDOWS_INFRA_CONTAINER="gcr.io/gke-release/pause-win:1.2.1"
+export WINDOWS_INFRA_CONTAINER="gcr.io/gke-release/pause-win:1.5.1"
```

Comment:
#98205 would you try to use the image with k8s.gcr.io/pause:3.4.1?

Is gcr.io/gke-release/pause-win:1.5.1 the official one?

Reply:
Yes, the pause-win image is stored in gke-release.